third_party/ti_simplelink_sdk/oad_merge_tool.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Texas Instruments Incorporated
```

Comment:
Since this is part of the CHIP SDK it should probably have the CHIP license blurb.

Should there be a `#!/usr/bin/env python` to make it executable directly?

Reply:
Make that `#!/usr/bin/env -S python -B` instead to avoid cache file creation.

Reply:
I'll update the copy and add the shebang. This isn't the file that creates the cache file, that's in the TI SDK and is run by ninja.